gpu/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# handlers file for gpu
+- name: Configure NVIDIA Fabric Manager
+  ansible.builtin.systemd:
+    name: nvidia-fabricmanager
```

Comment:
This package is only needed for specific GPU configurations. But we can address it later. I think there is no harm in having it installed anyway.